libs/SalesforceSDKCore/SalesforceSDKCoreTests/Utilities/TestDataAction.m

```diff
@@ -54,7 +54,8 @@ + (CSFNetwork*)mockNetworkWithAccount:(SFUserAccount *)account {
     NSURLSession *session = [self mockURLSession];
     
     // Create a mock network for this user's network
-    CSFNetwork *network = OCMClassMock([CSFNetwork class]);
+    CSFNetwork *networkToMock = [[CSFNetwork alloc] init];
+    CSFNetwork *network = OCMPartialMock(networkToMock);
```

Comment:
use partial mock instead of full mock, so that other APIs remain same(real) except the OCMStub defined ones below